tools/travis/coverage-pylint.sh

```diff
@@ -17,9 +17,9 @@ echo "$(pwd)"  # Know which directory tox is running this shell from.
 
 #coverage
 coverage erase
-coverage run --source=nltk $(pwd)/runtests.py -v --with-xunit
-coverage xml --omit=$(pwd)/*
-iconv -c -f utf-8 -t utf-8 nosetests.xml > nosetests_scrubbed.xml
+coverage run --source=nltk $(pwd)/runtests.py -v
```

Comment:
There's a [pytest plugin for coverage](https://github.com/pytest-dev/pytest-cov). Could you look into what we'd gain by using that? At work we seem to use it directly without having to install coverage.

Reply:
good idea, swapped to this